### PR TITLE
Release 2.13.0

### PR DESCRIPTION
**Pull request by Blackbaud-BrandonJones**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# 2.13.0 (2018-10-23)
+- Updated the [skyux-builder-plugin-stache](https://github.com/blackbaud/skyux-builder-plugin-stache/blob/master/CHANGELOG.md#151-2018-08-02) to `1.7.0`.
+  - This update includes the new `skyux stache-update` command to update their SKY UX, SKY UX Builder, and Stache dependencies for them, deleting node_modules and npm installing.
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: What do you think of changing this to "Adds the `skyux stache-update` command to update SKY UX, SKY UX Builder, and Stache dependencies. The command also deletes the `node_modules` folder and runs `npm install`."?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+- Style changes to the `stache-table-of-contents` component. [#512](https://github.com/blackbaud/stache2/pull/512)
+  - TOC now sits to the right of the content on full size views, and above the content on mobile views.
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: I'd change "TOC now sits" to "Updated TOC to place it" ... Also, hyphenate "full-size." ... And delete the comma.

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-LindseyRix) [reply CHANGELOG.md]: Much better.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+  - TOC now affixes to the top screen and follows users down the page as they scroll on full size views.
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: I'd change "TOC now affixes" to "Updated TOC to affix it." ... I think we need "of the" between "top" and "screen." ... And to go along with the first suggestion, I'd change "and follows" to follow." ... Also, hyphenate "full-size."

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
...
+  - TOC now gives a visual indication where the user is currently on the page by highlighting the current section being viewed on full size views.
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: I'd change "TOC now gives" to "Updated TOC to provide." ... I'd change "where the user is currently on the page" to "of the current location." I'd also delete "being viewed." ... Also, hyphenate "full-size."

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,11 @@
+# 2.13.0 (2018-10-23)
+- Updated the [skyux-builder-plugin-stache](https://github.com/blackbaud/skyux-builder-plugin-stache/blob/master/CHANGELOG.md#151-2018-08-02) to `1.7.0`.
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: I'd either delete "the" or add a noun after "skyux-builder-plugin-stache." Is this a dependency? If so, maybe "dependency" would work.

**Pull request merged**